cluster.yaml.example

```diff
@@ -179,29 +168,26 @@ kmsLivenessProbe:
   timeoutSeconds: 160
   failureThreshold: 3
   successThreshold: 1
+controllerManagerLivenessProbe:
+  initialDelaySeconds: 300
+  timeoutSeconds: 160
 portierisLivenessProbe:
```

Comment:
I don't see a `portierisLivenessProbe` in https://github.ibm.com/alchemy-containers/armada-openshift-master/blob/master/roles/hosted-master/templates/cluster.yaml.j2

Reply:
Good catch.  I'll follow-up with Bill.

Reply:
https://github.ibm.com/alchemy-containers/armada-deploy/issues/7648